pkg/support/support.go

```diff
@@ -219,10 +233,23 @@ func (bb *bundleBuilder) copyAndRedactFile(writer io.Writer, reader bufReader) e
 }
 
 func (bb *bundleBuilder) copyInFile(writer *zip.Writer, fileName string, prefix string) error {
-	// check filename validity?
-	f, err := bb.fs.Open(fileName)
-	if err != nil {
-		return err
+	var f io.Reader
+	if isService(fileName) {
+		service := strings.TrimPrefix(fileName, "service:")
+		cmd := bb.ecc.Create("journalctl", "-xu", service)
```

Comment:
Looks like we already have https://github.com/coreos/go-systemd in our go.mod file, and it uses a more official programmatic method of reading logs. Is it possible or worth it to use this library? I'm not sure since we're running finch on the host and go-systemd would need access to files or sockets (dbus) inside the Linux VM